apisix/init.lua

```diff
@@ -598,14 +598,26 @@ local function common_phase(phase_name)
     end
 
     if api_ctx.global_rules then
+        local orig_conf_type = api_ctx.conf_type
```

Comment:
`original? origin?`

Reply:
`orig_` is a common abbr for `original_`, for example:
https://github.com/apache/apisix/blob/15528d815716fa907abd595f989975f12932451a/apisix/core/table.lua#L109